flow/project.py

```diff
@@ -2127,6 +2131,10 @@ def export_job_stati(self, collection, stati):
             collection.update_one({'_id': status['job_id']},
                                   {'$set': status}, upsert=True)
 
+    def export_job_statuses(self, collection, statuses):
+        "Export the job statuses to a database collection."
+        self.export_job_stati(self, collection, statuses)
```

Comment:
I'm not 100% sure how `@deprecated` works, but I would guess that the implementation should be swapped. That is, `export_job_stati` should call `export_job_statuses` instead of the other way. Otherwise `export_job_statuses` will call a deprecated method and show a warning, right?

Reply:
I agree with your reasoning and will swap the flag around.